src/components/Textarea/Textarea.tsx

```diff
@@ -9,17 +9,41 @@ type Props = TextareaHTMLAttributes<HTMLTextAreaElement> & {
   autoFocus?: boolean
 }
 
-export const Textarea: FC<Props> = ({ autoFocus, ...props }) => {
+export const Textarea: FC<Props> = ({ autoFocus, maxLength, ...props }) => {
   const theme = useTheme()
   const ref = useRef<HTMLTextAreaElement>(null)
+  const [count, setCount] = useState(0)
```

Comment:
[MUST] Please calculate the initial value when `value` or `defaultValue` props is passed 🙏

Reply:
@nabeliwo Oh, I almost forgot🙇  fixed 🙏🙏